storybook/config.js

```diff
@@ -14,3 +14,6 @@ if (typeof window === 'object') {
 
 if (typeof window === 'object')
   window.__screener_storybook__ = require('@storybook/react').getStorybook()
+
+if (typeof window === 'object')
```

Comment:
This is a storybook bug. If I'm not careful it keeps automatically adding these 2 lines to this file.